src/sync/installation.ts

```diff
@@ -506,3 +514,10 @@ const updateRepo = async (subscription: Subscription, repoId: number, values: Re
 		Object.keys(rest).length && RepoSyncState.updateRepoFromSubscription(subscription, repoId, rest)
 	]);
 };
+
+const getBackfillSince = async (subscription: Subscription, data: BackfillMessagePayload): Promise<Date | null> => {
+	const commitSince = await getCommitSinceDate(data.jiraHost, NumberFlags.SYNC_MAIN_COMMIT_TIME_LIMIT, data.commitsFromDate);
```

Comment:
`CommitSince` should be fetched from the message payload. If backfilling takes more than a day, which will happen, then there is  a mismatch with the initial `CommitSince` which is used for backfilling

Reply:
--------
For restart backfill, there's no problem, the `data.commitsFromDate` is the date user picked from UI

----
For initial sync. The commitSince could be missing in the body
And I don't want to mess up with the msg body again, too much code changes. Maybe in another PR.

-------

So yes, this will be slightly different compare to the actual commit sync date, but it will be earlier, so it is "safe" from data missing point of view.

And current prod behaviour is already like that, the `commitsSince` day will changes everytime the task is reschdule again for initial sync.

Reply:
Let's create another PR for put dates inside the msg body to be more explicit.